### PR TITLE
Br cucumber return failed tests

### PR DESCRIPTION
**Pull request by mnalipatil**
hi sonali,

please review the code for failed Test Rerun configuration and suggest me any changes if required.

thank you,
Manali

### CODE REVIEW

File: src/test/java/com/qa/automation/runners/TestRunner.java [hunk 1]
```diff
@@ -9,10 +9,12 @@
 @CucumberOptions(
         features="classpath:features",//to tell cucumber where is ur feature file
         glue="com.qa.automation.stefdefs", // to tell cucumber where is ur step def code
-        tags="@LogoTitleValidation", // to tell which tagged feature file to execute
```
Comment (gawadesonali) [inline src/test/java/com/qa/automation/runners/TestRunner.java:12]: pleasee remove the tags. the "tags" should be empty.

**Review by gawadesonali — COMMENTED** [review]
Added the comments please check and revert it.

File: src/test/java/com/qa/automation/runners/TestRunner.java (hunk 1 above)
Comment (mnalipatil) [reply src/test/java/com/qa/automation/runners/TestRunner.java:12]: made the changes please check and approve the  PR

Comment (gawadesonali) [reply src/test/java/com/qa/automation/runners/TestRunner.java:12]: It Seems like the changes are good for now. approved.

**Pull request merged**